webpack/ForemanTasks/Components/TasksDashboard/__tests__/TasksDashboardSelectors.test.js

```diff
@@ -55,6 +64,10 @@ const fixtures = {
   'should select tasks-summary': () => selectTasksSummary(state),
   'should select tasks-summary when state is empty': () =>
     selectTasksSummary({}),
+  'should calcStoppedOther': () =>
+    calcStoppedOther(
+      state.foremanTasks.tasksDashboard.tasksSummary.stopped.by_result
+    ),
```

Comment:
I guess it would be fine to use:
```suggestion
    calcStoppedOther(selectTasksSummary(state)stopped.by_result)),
```

Reply:
or just passing `by_result` from fixtures :)